FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+Setup.swift

```diff
@@ -91,3 +142,25 @@ extension ComposeViewController {
             .store(in: &cancellable)
     }
 }
+
+// MARK: - NavigationChildController
+extension ComposeViewController: NavigationChildController {
+    func handleBackButtonTap() {
+        stopDraftTimer(withSave: false)
+
+        saveDraftIfNeeded(withAlert: true) { [weak self] error in
+            guard let self = self else { return }
+
+            if let error = error {
+                self.handle(error: error)
+            } else {
+                if var messageIdentifier = self.composeMessageService.messageIdentifier {
+                    messageIdentifier.draftMessageId = self.input.type.info?.id
+                    self.handleAction?(.update(messageIdentifier))
+                }
+                self.showToast("draft_saved".localized)
```

Comment:
This one is interesting. Do you show "draft_saved" after a draft was saved remotely or locally?

Reply:
On iOS we currently don't have local storage for drafts and messages. Alert is shown after remote save

Reply:
about this one. It seems you show this toast wrongly for some cases

>draft should not be saved if I only just rendered compose or reply. I have to actually input something first (any of the fields)

To test just open a compose screen and leave the screen without any actions. The same situation with reply/forward mode. 
You show a notification `showToast("draft_saved".localized)` but don't save a draft.